src/content/updates/migration/developers.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Migration
+tabs: ['Overview', 'Designers', 'Developers']
+---
+
+<AnchorLinks>
+
+- [Repo architecture](#repo-architecture)
+- [Migration tooling](#migration-tooling)
+- [Migration strategy](#migration-strategy)
+- [Carbon components](#carbon-components)
+- [Carbon elements](#carbon-elements)
+- [Feedback](#feedback)
+
+</AnchorLinks>
+
+## Repo architecture
+
+Two of Carbon's previous repos will be deprecated with v10: carbon-icons and carbon-themes. Iconography and themes in v10 are now handled as packages in the carbon-elements repo. 
+
+[@carbon/colors](https://github.com/IBM/carbon-elements/tree/master/packages/colors)
+
+[@carbon/themes](https://github.com/IBM/carbon-elements/tree/master/packages/themes)
```

Comment:
refers to icons and themes, but points to colors and themes.

Reply:
oops! good catch. fixed.